src/components/NetworkModal/index.tsx

```diff
@@ -58,27 +60,41 @@ export const ListItem = styled.div<{ selected?: boolean }>`
       `}
 `
 
-export const SelectNetworkButton = styled(ButtonEmpty)`
+export const SelectNetworkButton = styled(ButtonEmpty)<{ disabled?: boolean }>`
   background-color: transparent;
   color: ${({ theme }) => theme.primary};
   display: flex;
   justify-content: center;
   align-items: center;
-  &:focus {
-    text-decoration: none;
-  }
-  &:hover {
-    text-decoration: none;
-    border: 1px solid ${({ theme }) => theme.primary};
-  }
-  &:active {
-    text-decoration: none;
-  }
-  &:disabled {
-    opacity: 50%;
-    cursor: not-allowed;
-  }
+  ${({ disabled, theme }) =>
+    !disabled &&
+    `
+    &:focus {
+      text-decoration: none;
+    }
+    &:hover {
+      text-decoration: none;
+      border: 1px solid ${theme.primary};
+    }
+    &:active {
+      text-decoration: none;
+    }
+    &:disabled {
+      opacity: 50%;
+      cursor: not-allowed;
+    }
+  `}
+`
+
+const NewLabel = styled.div`
+  position: absolute;
+  top: 7px;
+  left: 93px;
+  font-size: 8px;
+  font-weight: 500;
+  color: #ff537b;
```

Comment:
hard code?

Reply:
What's issue

Reply:
This one not export anywhere, only use here

Reply:
I cloned it from [here](https://github.com/KyberNetwork/kyberswap-interface/blob/main/src/components/YieldPools/styleds.tsx#L143-L150) with some customized